src/vue/common/OperationDetails/Issuance.Details.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="details">
+    <table class="details__table">
+      <tr>
```

Comment:
I'm not sure it's a good idea to use the table tag, but even so, you can easily reduce the code by wrapping the common parts to some `detail` component

```
<detail :label="'..." :value="..." />
<detail :label="'..." :value="..." />
<detail :label="'..." :value="..." />

```

Reply:
I’d pref the simplest way, `detail` isn’t such good solution either since it sucks in long strings and not so flexible in label’s width. tables, that we got right now, lgtm.